contracts/services/astroport-lper/src/tests.rs

```diff
@@ -688,23 +688,42 @@ fn provide_single_sided_liquidity_cw20_lp_token() {
 fn test_limit_single_sided_liquidity() {
     let setup = LPerTestSuite::default();
     let wasm = Wasm::new(&setup.inner.app);
+    let bank = Bank::new(&setup.inner.app);
 
-    let error = wasm
-        .execute::<ExecuteMsg<ActionsMsgs, OptionalServiceConfig>>(
-            &setup.lper_addr,
-            &ExecuteMsg::ProcessAction(ActionsMsgs::ProvideSingleSidedLiquidity {
-                asset: setup.inner.pool_asset1.clone(),
-                limit: Some(Uint128::new(1)),
-                expected_pool_ratio_range: None,
-            }),
-            &[],
-            setup.inner.processor_acc(),
-        )
-        .unwrap_err();
+    let query_balance = |address: &str| -> u128 {
```

Comment:
Really should be a helper function somewhere

Reply:
changing it into the other PR since I have conflicts for this